src/Rector/v8/v6/RefactorTCARector.php

```diff
@@ -167,21 +167,25 @@ private function addFieldControlInsteadOfWizardsAddListEdit(Array_ $items): void
                 continue;
             }
 
-            foreach ($fieldValue->value->items as $configValue) {
-                if (null === $configValue) {
+            $fieldValueArray = $fieldValue->value;
+            foreach ($fieldValueArray->items as $configValue) {
+                if (! $configValue instanceof ArrayItem) {
```

Comment:
If we have `SomeType|null`, the exact match to `! not SomeType` will make IDE and phpstan smarted. If this condition passes, it can be only `SomeType`

vs `$value !== null` means only that value is anything except null - weaker